docs/geedocs/answer/4492599.html

```diff
@@ -90,7 +93,7 @@
 <p><strong>Browsers</strong></p>
 <p>Google Earth Enterprise is supported on the following browsers on all operating systems:</p>
 <ul>
-<li dir="ltr">Google Chrome version 10 and later (32-bit)</li>
+<li dir="ltr">Google Chrome version 10 and later</li>
```

Comment:
I wrote ticket #310 to set up a testing framework.  I also updated the list of supported browsers.